update.check/src/test/java/org/kaazing/gateway/update/check/UpdateCheckWithRCVersionIT.java

```diff
@@ -41,7 +43,14 @@
         {
             GatewayConfiguration configuration = createGatewayConfiguration();
             //The product information from the server JAR will be overridden by the call to reset
-            VersionUtils.reset("Gateway", "Kaazing WebSocket Gateway", "5.6.1-RC001", "");
+           // ProductInfoReader.reset("Gateway", "Kaazing WebSocket Gateway", "5.6.1-RC001", "");
+            ProductInfo p = new ProductInfo();
+            p.setEdition("Gateway");
+            p.setTitle("Kaazing WebSocket Gateway");
+            p.setVersion("5.6.1-RC001");
```

Comment:
@DoruM (CC: @dpwspoon) 
I just wanted to understand why are we hardcoding `5.6.1-RC001` version here, or for that matter why would we hard code any GW version?